### ISSUE TITLE
hello-world comilation fails with cannot satisfy -package-id citeproc-0.5...

### ISSUE THREAD

**Issue by andrewufrank** [agda/agda#5693]
I am trying to use agda for the first time. installed from debian (bullseye). the hello-world example failed with an error which seems already to be reported. 
i then installed with `cabal install Agda` and got  (2.6.2.1)
copied the library following the instructions - including the setup in the `.agda` directory.

with `agda --compile hello-world.agda`

the library is found but I get the error

```
Calling: ghc -O -o /home/frank/agda_folder/hello/hello-world -Werror -i/home/frank/agda_folder/hello -main-is MAlonzo.Code.QhelloZ45Zworld /home/frank/agda_folder/hello/MAlonzo/Code/QhelloZ45Zworld.hs --make -fwarn-incomplete-patterns -fno-warn-overlapping-patterns
Loaded package environment from /home/frank/.ghc/x86_64-linux-8.10.4/environments/default
Compilation error:
<command line>: cannot satisfy -package-id citeproc-0.5-9f761cb4a516ae7cf8a09a7b5b9aa54b8a98e0ffc680db70cd56fa10d443ba33
    (use -v for more information)


```

I cannot see what is wrong and it is frustrating when the first example copied from instructions fails. thank you for help!

**Comment by gallais** [comment]
> it is frustrating when the first example copied from instructions fails

It is indeed. I suspect this is once again #4627.

**Comment by andreasabel** [comment]
Are we using `citeproc`?  I don't think so. @andrewufrank wrote:
```
> Loaded package environment from /home/frank/.ghc/x86_64-linux-8.10.4/environments/default
```
Maybe your default package environment is broken.  You might have to remove entries there.
I ran into similar problems in 
- https://github.com/haskell/cabal/issues/7820.

**Comment by nad** [comment]
If #4627 was fixed this kind of problem would presumably not matter.

**Comment by andrewufrank** [comment]
thank you for your interest and helpful hints. after reading your pointers I realized that the problem is probably not with agda but with the (well-known unfortunate) problems with cabal and stack. 

I have a ghc environment (using Haskell for around 30 years) and given that `citeproc` is not used by `agda` (but in some programs I use) I built my program new (to make sure there are no issues created by the `cabal update` used). 

Then deleted in `.ghc` the `environment` folder. I assume it was the `default` file there, which included a reference to `citeproc`. I deleted also the `dist-newstyle` and the `MAlonzo` folder. Now things work as advertised. 

Thank you for producing agda!
